Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1609,6 +1609,11 @@ void WebProcess::prepareToSuspend(bool isSuspensionImminent, MonotonicTime estim
         platformMediaSessionManager->processWillSuspend();
 #endif
 
+#if !ENABLE(WEBPROCESS_CACHE)
+    if (!m_suppressMemoryPressureHandler)
```

Comment:
It is confusing to add new code and apparently change behavior in a patch that is meant to simply introduce a new build flag.

Reply:
That's a good point, Chris! I had to re-introduce this code, since it was removed by https://commits.webkit.org/266267@main.

Thanks for reviewing!